### PR TITLE
Adding additional cache flags for query context

### PR DESCRIPTION
**Pull request by acdn-abatos**

### CODE REVIEW

File: queries.go [hunk 1]
```diff
@@ -28,9 +28,11 @@ const (
...
 // Context constants
 const (
-	TIMEOUT          = "timeout"
-	SKIPEMPTYBUCKETS = "skipEmptyBuckets"
-	QUERYID          = "queryId"
+	TIMEOUT                  = "timeout"
+	SKIPEMPTYBUCKETS         = "skipEmptyBuckets"
+	QUERYID                  = "queryId"
+	POPULATECACHE            = "populateCache"
+        POPULATERESULTLEVELCACHE = "populateResultLevelCache"
```
Comment (jaimeyu) [inline queries.go:35]: Weird white space issue, can you run go fmt first. I have a suspicion we'll have a weird white space commit in a future PR

**Review by jaimeyu — COMMENTED** [review]
Are these variables being used somewhere? Strange this commit is only to define them

File: queries.go (hunk 1 above)
Comment (acdn-abatos) [reply queries.go:35]: Strange in the git repo and in my local repo it seems fine

Comment (jaimeyu) [reply queries.go:35]: Must be the tag commit that fixed it. Weird it shows up here.

Comment (acdn-abatos) [reply queries.go:35]: I made another pull request that should address it

**Pull request merged**